packages/client/src/styles.less

```diff
@@ -134,7 +134,7 @@ div.rating {
         height: 24px;
         width: 24px;
     }
-    &.large {
+    &.medium {
```

Comment:
had duplicate larges here. this may result in `large`s elsewhere being too big now, because I think they may have been short-circuting on what is now `medium`.